stdlib/builtins.pyi

```diff
@@ -472,14 +472,12 @@ class bytes(ByteString):
     def zfill(self, __width: int) -> bytes: ...
     @classmethod
     def fromhex(cls, __s: str) -> bytes: ...
-    @classmethod
-    def maketrans(cls, frm: bytes, to: bytes) -> bytes: ...
+    @staticmethod
+    def maketrans(__frm: bytes, __to: bytes) -> bytes: ...
     def __len__(self) -> int: ...
     def __iter__(self) -> Iterator[int]: ...
     def __str__(self) -> str: ...
     def __repr__(self) -> str: ...
-    def __int__(self) -> int: ...
```

Comment:
I wonder if these were added on purpose to make bytes fulfill the `SupportsInt` and `SupportsFloat` protocols. `int(b'12')` works at runtime. Probably safer to keep them.

Reply:
1. no mypy primer diffs
2. the int construct specifically gets bytes (so does float) and treats it as a string
3. Other functions that receive SupportsInt fail on bytes - itertools.count(b'1',1)

Reply:
Sounds good, I will merge